tests/net/dhcpv4/client/src/main.c

```diff
@@ -36,8 +36,8 @@ LOG_MODULE_REGISTER(net_test, CONFIG_NET_DHCPV4_LOG_LEVEL);
 #define NET_LOG_ENABLED 1
```

Comment:
One minor nit, please change the commit subject of the test patch to `tests: net: dhcpv4: ....` so that it is easier to find when grepping the git log.